### PR TITLE
[no release notes] Cleanup release notes

### PR DESCRIPTION
**Pull request by fsamuel-bs**
**Goals (and why)**: Cleanup 0.46 release notes.

**Implementation Description (bullets)**: -

**Concerns (what feedback would you like?)**: -

**Where should we start reviewing?**: -

**Priority (whenever / two weeks / yesterday)**: -

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/2119)

### CODE REVIEW

File: docs/source/release_notes/release-notes.rst [hunk 1]
```diff
@@ -151,6 +121,15 @@ develop
...
+           If a single leader is configured, it will no longer go via HTTPS/Jetty to request timestamps and locks from itself.
+           Aside from a minor perf improvement, this should also fix a potential livelock/deadlock when the leader is under heavy load,
+           where previously the Jetty threadpool could become full of requests awaiting timestamps, preventing any timestamp requests from being serviced, and requiring a server reboot to resolve.
+
+           We recommend HA clusters under heavy load switch to using a standalone timestamp service, as they may also be vulnerable to this failure mode.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2091>`__)
+
+    *    - |improved|
          - The dropwizard independent implementation of the TimeLock server has been separated into a new project, ``timelock-impl``.
            This should not affect users directly, unless they depended on classes from within the TimeLock server.
```
Comment (jeremyk-91) [inline docs/source/release_notes/release-notes.rst]: Hmm, this comment sounds like it's implying dev break (which it is).

File: docs/source/release_notes/release-notes.rst [hunk 2]
```diff
@@ -162,7 +141,26 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2093>`__)
 
     *    - |fixed|
-         - Added backwards compatibility for passing row values into AtlasConsole functions
+         - AtlasDB clients now retry lock requests if the server lose leadership while the request is blocking.
+           In the past, this scenario would cause 500 errors that were not retried by the client.
```
Comment (jeremyk-91) [inline docs/source/release_notes/release-notes.rst]: Should probably also say that the server now returns 503s (relevant for people that use lock service on their own?)

Also nit - if the server _loses_ leadership

File: docs/source/release_notes/release-notes.rst [hunk 3]
```diff
@@ -134,11 +99,16 @@ develop
          - Sweep now dynamically adjusts the number of (cell, ts) pairs across runs:
 
            - On a failure run, sweep halves the number of pairs to read and to delete on subsequent runs.
-           - On a success run, sweep slowly increases the number of pairs to read and to delete on subsequent runs, up to a configurable maximum.
+           - On a success run, sweep slowly increases the number of (cell, ts) pairs to read and to delete on subsequent runs, up to a configurable maximum.
 
            This should fix the issue where we were unable to sweep cells with a high number of mutations.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2060>`__)
 
+    *    - |changed|
```
Comment (jeremyk-91) [inline docs/source/release_notes/release-notes.rst]: Might be cheeky, but "improved"?

**Comment by fsamuel-bs** [comment]
@jeremyk-91: usually I'd agree, but releasing Background Sweep seems to be the most impactful change of this release, and many of the following changes are related to it, so I thought it should go first. We can change it later if needed.

**Pull request merged**